src/System.Buffers/ref/System.Buffers.cs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+// ------------------------------------------------------------------------------
+// Changes to this file must follow the http://aka.ms/api-review process.
+// ------------------------------------------------------------------------------
+
+namespace System.Buffers
+{
+    public abstract class ArrayPool<T>
+    {
+        public const int DefaultNumberOfArraysPerBucket = 50;
+        public const int DefaultArrayLength = 1024000;
+
+        public static ArrayPool<T> Shared{ get; }
+        public static ArrayPool<T> Create(int maxArrayLength = DefaultArrayLength, int arraysPerBucket = DefaultNumberOfArraysPerBucket);
+        public abstract int RentedBufferCount { get; }
+        public abstract int TotalBufferCount { get; }
```

Comment:
What's the value of exposing these members?  It adds interlocked operations to each operation, and I'm not sure what value these provide other than diagnostic.